lib/plugins/aws/deployFunction/index.js

```diff
@@ -132,10 +134,18 @@ class AwsDeployFunction {
 
     if (functionObj.vpc || providerObj.vpc) {
```

Comment:
`functionObj.vpc` might be undefined and that caused a problem in line `135` (`functionObj.vpc.securityGroupIds`)